filebeat/tests/system/test_modules.py

```diff
@@ -158,26 +167,22 @@ def _test_expected_events(self, module, test_file, objects):
         for ev in expected:
             found = False
             for obj in objects:
-                # For Modules not in EXTENDED_COMPARE on module level fields are compared
-                if module not in EXTENDED_COMPARE:
-                    if ev["_source"][module] == obj[module]:
-                        found = True
-                        break
 
                 # Flatten objects for easier comparing
                 obj = self.flatten_object(obj, {}, "")
-                ex = self.flatten_object(ev["_source"], {}, "")
-
                 clean_keys(obj)
-                clean_keys(ex)
 
-                # All modules in EXTENDED_COMPARE are checked in more detail
-                if ex == obj:
+                # Remove timestamp for comparison where no timestamp is not part of the log line
```

Comment:
there is a typo in this sentence, `where **no** timestamp is not part...`

Reply:
fixed